### PR TITLE
Update the blob storage adapter.

### PR DESCRIPTION
**Pull request by ykobuk**

### CODE REVIEW

File: odd_collector_azure/adapters/blob_storage/mapper/container.py [hunk 1]
```diff
@@ -42,10 +42,10 @@ def map_file(file: File, generator: BlobGenerator) -> DataEntity:
 
 
 def map_folder(
-    folder: Folder, generator: BlobGenerator
+    folder: Folder, generator: AzureBlobStorageGenerator
 ) -> tuple[str, deque[DataEntity]]:
     container, *keys = folder.path.split("/")
```
Comment (Vixtir) [inline odd_collector_azure/adapters/blob_storage/mapper/container.py:47]: Same as above

**Review by Vixtir — CHANGES_REQUESTED** [review]

File: odd_collector_azure/adapters/blob_storage/adapter.py [hunk 2]
```diff
@@ -1,45 +1,47 @@
...
 
+    def create_generator(self) -> Generator:
+        return AzureBlobStorageGenerator(
+            azure_cloud_settings={
+                'account_name': self.config.account_name,
+                'container_name': self.config.dataset_config.container
+            }
+        )
+
     def get_data_source_oddrn(self) -> str:
```
[the remark this answers is not in the document]
Comment (ykobuk) [reply odd_collector_azure/adapters/blob_storage/adapter.py]: Done.

File: odd_collector_azure/adapters/blob_storage/mapper/container.py [hunk 3]
```diff
@@ -11,9 +11,9 @@
 from .column import map_columns
 
 
-def map_file(file: File, generator: BlobGenerator) -> DataEntity:
+def map_file(file: File, generator: AzureBlobStorageGenerator) -> DataEntity:
     container, *keys = file.path.split("/")
```
[the remark this answers is not in the document]
Comment (ykobuk) [reply odd_collector_azure/adapters/blob_storage/mapper/container.py]: Done.

File: odd_collector_azure/adapters/blob_storage/mapper/container.py (hunk 1 above)
Comment (ykobuk) [reply odd_collector_azure/adapters/blob_storage/mapper/container.py]: Done.

**Review by Vixtir — APPROVED** [review]

**Pull request merged**